src/Compilers/CSharp/Portable/CodeGen/Optimizer.cs

```diff
@@ -1525,9 +1547,10 @@ private void RecordVarWrite(LocalSymbol local)
             locInfo.LocalDefs.Add(locDef);
         }
 
-        private static bool CanScheduleToStack(LocalSymbol local)
+        private bool CanScheduleToStack(LocalSymbol local)
```

Comment:
I'm confused about this terminology. What does it mean to "schedule to stack"? Is there any literature on this optimization?